tests/test_filetransfer.py

```diff
@@ -375,7 +375,7 @@ def test_upload(self):
                              {target[0], target[2]})
             self.assertEqual(set(os.listdir(CLIENT_DIR)),
                              {target[0], target[2]})
-            self.assertEqual(os.listdir(FILES_DIR), FILES)
+            self.assertEqual(set(os.listdir(FILES_DIR)), set(FILES))
```

Comment:
Is it failing because of the ordering? `self.assertItemsEqual()` might be useful in that case.

Reply:
Didn't know of `self.assertItemsEqual()` :)